docs/usage/advanced/immutable-backup-buckets.md

```diff
@@ -0,0 +1,96 @@
+# Immutable Backup Buckets
+---
+title: Immutable Backup Buckets
+description: Configuration guide for enabling immutable etcd backup buckets across different cloud providers
+---
+
+## Overview 
+
+Immutable backup buckets ensure that etcd backups cannot be modified or deleted after creation, leveraging immutability features provided by supported cloud providers. This is essential for meeting security, compliance, and operational requirements.
+
+> ℹ️ For background, see [Gardener Issue #10866](https://github.com/gardener/gardener/issues/10866).
+
+## Core Concepts
+
+Gardener supports immutable backup buckets for `AWS`, `GCP`, and `Azure` via their respective provider extensions. When enabled, the extension will:
+
+- **Create** buckets with the specified immutability policy if they do not exist.
+- **Reconcile** existing buckets to enforce the desired immutability policy.
+- **Prevent** reducing the retention period or disabling immutability once the policy is locked.
+- **Apply** lifecycle-based delayed deletion on `GCP` and `Azure` when immediate deletion is not permitted.
+
+> **Note:** Once immutability is locked, it cannot be disabled or shortened.
+
+## Admission Webhook Enforcement
+
+An admission webhook ensures:
+
+- **Locking:** Once `locked: true`, immutability cannot be disabled.
+- **Retention:** Once the retention period is set and locked, it cannot be decreased.
+
+## Configuration
+
+To enable immutable backup buckets, specify the immutability settings in `.spec.backup.providerConfig`. The Gardenlet will automatically create or update the bucket according to the configuration.
+
+- See [BackupBucket Resource Contract](../../extensions/resources/backupbucket.md)
+- See [Gardenlet Controller Concepts – BackupBucket Controller](../../concepts/gardenlet.md#backupbucket-controller)
+
+## Example Configurations
+
+### `GCP`
+
+```yaml
+apiVersion: gcp.provider.extensions.gardener.cloud/v1alpha1
+kind: BackupBucketConfig
+metadata:
+  name: example-immutable-backup
+spec:
+  retentionPeriod: 96h        # Backups are immutable for 4 days
+  retentionType: bucket       # Bucket-level immutability
+  locked: false               # Set to true to lock the policy
+```
+
+> `GCP` uses lifecycle rules to delay object deletion until the retention period expires. Once `locked: true` is set, the policy cannot be reverted or shortened. See [`GCP` Extension Usage – BackupBucketConfig](https://github.com/gardener/gardener-extension-provider-gcp/blob/master/docs/usage/usage.md#backupbucketconfig)
+
+### `AWS`
+
+```yaml
+apiVersion: aws.provider.extensions.gardener.cloud/v1alpha1
+kind: BackupBucketConfig
+immutability:
+  retentionType: bucket  # Enable S3 Object Lock on bucket
+  retentionPeriod: 96h  # Backups are immutable for 4 days
+  mode: compliance     # or governance
+
```

Comment:
```suggestion
```yaml
apiVersion: aws.provider.extensions.gardener.cloud/v1alpha1
kind: BackupBucketConfig
metadata:
  name: example-immutable-backup
immutability:
  retentionType: bucket  # Enable S3 Object Lock on bucket
  retentionPeriod: 96h  # Backups are immutable for 4 days
  mode: compliance     # or governance
```

```